lib/jekyll/collection.rb

```diff
@@ -170,7 +170,8 @@ def write?
     #
     # Returns the URL template to render collection's documents at.
     def url_template
-      metadata.fetch('permalink', "/:collection/:path:output_ext")
+      default = Utils.append_permalink_ending("/:collection/:path", site.permalink_style)
+      metadata.fetch('permalink', default)
```

Comment:
and now I've reached the limit of my ruby knowledge :) I'm not sure I know what that would look like.